src/json.erl

```diff
@@ -0,0 +1,21 @@
+-module(json).
```

Comment:
does json parsing library parse into a nested json map?

Aslo why atoms instead of strings for keys?

```suggestion
-module(potato_json).
```

Reply:
it parses into maps into maps, lists into lists, etc; recursively.

erlang is weird about string representation, it is either binary, atoms, or lists; and i haven't decided which one i like better

potato_json is too much to type ...